.gitpod.yml

```diff
@@ -0,0 +1,14 @@
+image: gitpod/workspace-go
+tasks:
+  - init: go mod tidy  
```

Comment:
Should we really want to tidy or just get the dependencies instead?

Reply:
I'm OK either way - I figure this is essentially low/no risk given how closely we manage dependencies anyway, and dependabot does some of this already anyway.